### ISSUE TITLE
No org.claws_mail.Claws-Mail.Debug to install debugging symbols

### ISSUE THREAD

**Issue by crass** [flathub/org.claws_mail.Claws-Mail#19]
In trying to debug #18 I noticed that there's no debug symbols package, which could give better gdb backtraces and help with debugging crashes.

**Comment by cobratbq** [comment]
I will look into debugging symbols. I need to figure out how to package those.

**Comment by cobratbq** [comment]
Okay, I discovered an inconsistency. Please try installing debug symbols through package `org.claws_mail.Claws_Mail.Debug`. Although you might still be missing some debug symbols, though maybe not an issue for your case.

I am not sure why the naming is inconsistent. Check `flatpak remote-info -m flathub org.claws_mail.Claws-Mail` to view for yourself which extensions there are and what names they have.

**Comment by cobratbq** [comment]
@crass the test build in comment https://github.com/flathub/org.claws_mail.Claws-Mail/pull/20#issuecomment-926896111 allows you to install a package with the additional debug symbols. This is assuming that your issues persist with package `3.18.0`, of course.

**Comment by crass** [comment]
I've been able to install the debug symbols as you've suggested by changing `Claws-Mail` to `Claws_Mail` from the main flathub repository. 

> @crass the test build in comment [#20 (comment)](https://github.com/flathub/org.claws_mail.Claws-Mail/pull/20#issuecomment-926896111) allows you to install a package with the additional debug symbols. This is assuming that your issues persist with package `3.18.0`, of course.

I'm confused by this comment because I didn't need to install from the testing repository. Maybe because I waited sufficiently long enough for it to be rolled out to the main repo?

I've also noticed that it seems that this only includes debug symbols for the Claws-Mail binary, but not any of the libraries it uses. For instance, libetpan has no debug symbols. And another issue that I haven't figured out yet (which is more of a flatpak general issue) is how to get debugging symbols for libc.

**Comment by cobratbq** [comment]
> I'm confused by this comment because I didn't need to install from the testing repository. Maybe because I waited sufficiently long enough for it to be rolled out to the main repo?

That confusion is my bad: I did not update the version of the flatpak-package with the upgrade to Claws-Mail 3.18.0. The last update (the broken one, and also the most recent one) were Claws-Mail 3.18.0 with flatpak-package saying 3.17.8. I still need to fix that.

> I've also noticed that it seems that this only includes debug symbols for the Claws-Mail binary, but not any of the libraries it uses. For instance, libetpan has no debug symbols. And another issue that I haven't figured out yet (which is more of a flatpak general issue) is how to get debugging symbols for libc.

I _think_ the solution is two-fold:

1. #20 removes the `/lib/debug` path from clean-up instructions. That should preserve a lot more debug symbols. I just found out how flatpak handles debugging (extensions), so fixing this in the build instructions now.
2. According to [this documentation on flatpak](https://docs.flatpak.org/en/latest/debugging.html) it might be helpful to install the SDK (`org.freedesktop.Sdk/x86_64/20.08`, but can also be discovered through `flatpak info -m org.claws_mail.Claws-Mail`)

As for the specific issue: I suspect (but have not confirmed yet) that recent commits in `libetpan` are causing this.

@crass thanks for your effort. You've pointed me in the right direction on a couple of things.
